src/dlgPackageExporter.cpp

```diff
@@ -662,228 +665,231 @@ void dlgPackageExporter::slot_export_package()
         }
     }
 
+    qDebug() << stagingDirName << mPackagePathFileName << mXmlPathFileName << mPackageName << mPackageConfig << isOk;
```

Comment:
It would be more use if there was some introducers between the items:
```suggestion
    qDebug().nospace().noquote() << "Staging dir: " << stagingDirName << ", Package pathFileName: " << mPackagePathFileName << ", XmlPathFileName: " << mXmlPathFileName << ", Package name:" << mPackageName << ", Package Config: " << mPackageConfig << ", StatusOkay: " << isOk;
```